tmt/base.py

```diff
@@ -4658,6 +4659,21 @@ def show_runner(self, logger: tmt.log.Logger) -> None:
         for _, key_formatted, value_formatted in self.runner.facts.format():
             log(key_formatted, value_formatted, shift=1)
 
+    def copy_scripts(self) -> None:
+        """
+        Copy the tmt helper scripts under the running workdir
+        into a new scripts directory.
+        """
+        assert self.workdir is not None
+        scripts = tmt.steps.scripts.SCRIPTS
+        destination = self.workdir / "scripts"
```

Comment:
```suggestion
        destination = self.workdir / tmt.steps.scripts.SCRIPTS_DIR_NAME
```

The `scripts` constant is used across multiple places, let's define it once, in the `scripts` module and use it here and in the `local.py` as well.